OVERVIEW.md

```diff
@@ -92,6 +92,34 @@ sub = pubsub.subscription "my-topic-sub"
 msgs = sub.pull
 ```
 
+# DNS
+
+[Google Cloud DNS](https://cloud.google.com/dns/) ([docs](https://cloud.google.com/dns/docs)) is a high-performance, resilient, global DNS service that provides a cost-effective way to make your applications and services available to your users. This programmable, authoritative DNS service can be used to easily publish and manage DNS records using the same infrastructure relied upon by Google. To learn more, read {What is Google Cloud DNS?}[https://cloud.google.com/dns/what-is-cloud-dns].
```

Comment:
How come some links use `[]()` and the What is Cloud DNS uses `{}[]` ? Is that a typo or intended?